proposals/compatibility-mode.md

```diff
@@ -89,6 +89,10 @@ A bind group may not reference a subset of array layers. Only views of the entir
 
 **Justification**: OpenGL ES does not support texture views.
 
+### 7. Disallow `sample_mask` builtin in WGSL.
```

Comment:
We should say whether you can't use it at all (it fails at createShaderModule) or you just can't create pipelines that use it. I know at some point in the past, I was involved in some discussion about this. I prefer having the error at pipeline creation time because (1) it lets you write WGSL code that mixes both compat and non-compat code and use it in compat, and (2) all compat implementations are expected to have the same WGSL parsing code of the full implementation.

Reply:
I'm ok with deferring errrors until pipeline creation time. I've added some text to that effect.